samples/flip-coin/README.md

```diff
@@ -0,0 +1,52 @@
+# The flip-coin pipeline
+
+The [flip-coin pipeline](https://github.com/kubeflow/pipelines/blob/master/samples/core/condition/condition.py)
+is used to demonstrate the use of conditions.
+
+* [Installing Tekton Pipelines on Kubernetes](https://github.com/tektoncd/pipeline/blob/master/docs/install.md#installing-tekton-pipelines-on-kubernetes) 
+
+* [Installing Tekton Pipelines on OpenShift](https://github.com/tektoncd/pipeline/blob/master/docs/install.md#installing-tekton-pipelines-on-openshift)
+
+There are two ways to generate `Tekton` YAML. One is using `dsl-compile-tekton` command line tool, another way is using `TektonCompiler` inside your pipeline script. Both will generate the same `Tekton` YAML file. Then apply the YAML to your cluster by using the `tkn` command.
+
+* Using command line to generate YAML
+
+    ```
+    # Using kfp-tekton command line tool to generate `Tekton` YAML
+    dsl-compile-tekton --py condition.py --output condition.yaml
+    ```
+
+* Using `TektonCompiler` to generate YAML
+    ```
+    # Replace the kfp compile code in the condition.py's `main`: 
+    # From
+    #   `kfp.compiler.Compiler().compile(flipcoin_pipeline, __file__ + '.yaml')`
+    # To
+    #   `from kfp_tekton.compiler import TektonCompiler`
+    #   `TektonCompiler().compile(flipcoin_pipeline, __file__.replace('.py', '.yaml'))`
```

Comment:
I'm not sure do we really need these comments since we already replaced for them in our .py file. Having this might confuse the users.

Reply:
ok, sure, I will remove this.